website/content/backend-server.md

```
@@ -1,199 +1,199 @@
+| **Admin Interface**   | django-unfold                                   | 0.65    |
+| **Package Manager**   | Poetry                                          | 2.0+    |
+| **Python**            | 3.11+                                           | --      |
+| **Linting**           | ruff, black, isort                              | --      |
+
+---
+
+## Application Structure
+
+The server code lives in the `server/` directory of the repository:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)